### DISCUSSION TITLE
Vulnerablities are always empty.

### DISCUSSION THREAD

**Discussion by Premfer** [quay/clair#1372] [Q&A] [answered]
Hi when I scan images with clair the vulnerabilities seems to be empty always for all images. 
Clair version: quay.io/projectquay/clair:4.1.5
I saw this following log , not sure if this is related. Let me know is there something need to be checked or updated or how to proceed.

> {"level":"error","component":"libvuln/updates/Manager.Start","error":"updating errors:\noracle-2011-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2011.xml.bz2\": x509: certificate signed by unknown authority\noracle-2012-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2012.xml.bz2\": x509: certificate signed by unknown authority\noracle-2014-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2014.xml.bz2\": x509: certificate signed by unknown authority\noracle-2008-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2008.xml.bz2\": x509: certificate signed by unknown authority\noracle-2010-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2010.xml.bz2\": x509: certificate signed by unknown authority\noracle-2007-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2007.xml.bz2\": x509: certificate signed by unknown authority\noracle-2013-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2013.xml.bz2\": x509: certificate signed by unknown authority\noracle-2020-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2020.xml.bz2\": x509: certificate signed by unknown authority\nubuntu-xenial-updater: failed to retrieve OVAL database: Get \"https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.xenial.cve.oval.xml.bz2\": x509: certificate signed by unknown authority\noracle-2015-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2015.xml.bz2\": x509: certificate signed by unknown authority\nubuntu-focal-updater: failed to retrieve OVAL database: Get \"https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.focal.cve.oval.xml.bz2\": x509: certificate signed by unknown authority\noracle-2009-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2009.xml.bz2\": x509: certificate signed by unknown authority\nubuntu-eoan-updater: failed to retrieve OVAL database: Get \"https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.eoan.cve.oval.xml.bz2\": x509: certificate signed by unknown authority\noracle-2016-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2016.xml.bz2\": x509: certificate signed by unknown authority\nalpine-main-v3.11-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.11/main.json\": x509: certificate signed by unknown authority\noracle-2017-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2017.xml.bz2\": x509: certificate signed by unknown authority\nalpine-main-v3.8-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.8/main.json\": x509: certificate signed by unknown authority\noracle-2018-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2018.xml.bz2\": x509: certificate signed by unknown authority\nalpine-main-v3.6-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.6/main.json\": x509: certificate signed by unknown authority\noracle-2019-updater: Get \"https://linux.oracle.com/security/oval/com.oracle.elsa-2019.xml.bz2\": x509: certificate signed by unknown authority\nalpine-main-v3.4-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.4/main.json\": x509: certificate signed by unknown authority\nalpine-community-v3.7-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.7/community.json\": x509: certificate signed by unknown authority\nalpine-main-v3.12-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.12/main.json\": x509: certificate signed by unknown authority\nalpine-main-v3.10-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.10/main.json\": x509: certificate signed by unknown authority\nalpine-main-v3.9-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.9/main.json\": x509: certificate signed by unknown authority\nalpine-main-v3.5-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.5/main.json\": x509: certificate signed by unknown authority\nalpine-community-v3.12-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.12/community.json\": x509: certificate signed by unknown authority\nalpine-community-v3.10-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.10/community.json\": x509: certificate signed by unknown authority\nalpine-main-v3.7-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.7/main.json\": x509: certificate signed by unknown authority\nalpine-main-v3.3-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.3/main.json\": x509: certificate signed by unknown authority\nalpine-community-v3.6-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.6/community.json\": x509: certificate signed by unknown authority\nalpine-community-v3.5-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.5/community.json\": x509: certificate signed by unknown authority\naws-linux2-updater: failed to create client: failed to make request for mirrors: Get \"https://cdn.amazonlinux.com/2/core/latest/x86_64/mirror.list\": x509: certificate signed by unknown authority\nalpine-community-v3.11-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.11/community.json\": x509: certificate signed by unknown authority\nalpine-community-v3.9-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.9/community.json\": x509: certificate signed by unknown authority\nalpine-community-v3.8-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.8/community.json\": x509: certificate signed by unknown authority\nalpine-community-v3.4-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.4/community.json\": x509: certificate signed by unknown authority\nalpine-community-v3.3-updater: alpine: error making request: Get \"https://secdb.alpinelinux.org/v3.3/community.json\": x509: certificate signed by unknown authority\ndebian-wheezy-updater: failed to retrieve OVAL database: Get \"https://www.debian.org/security/oval/oval-definitions-wheezy.xml\": x509: certificate signed by unknown authority\ndebian-stretch-updater: failed to retrieve OVAL database: Get \"https://www.debian.org/security/oval/oval-definitions-stretch.xml\": x509: certificate signed by unknown authority\ndebian-jessie-updater: failed to retrieve OVAL database: Get \"https://www.debian.org/security/oval/oval-definitions-jessie.xml\": x509: certificate signed by unknown authority\npyupio: Get \"https://github.com/pyupio/safety-db/archive/master.tar.gz\": x509: certificate signed by unknown authority\nphoton-updater-photon2: Get \"https://packages.vmware.com/photon/photon_oval_definitions/com.vmware.phsa-photon2.xml\": x509: certificate signed by unknown authority\nphoton-updater-photon1: Get \"https://packages.vmware.com/photon/photon_oval_definitions/com.vmware.phsa-photon1.xml\": x509: certificate signed by unknown authority\nphoton-updater-photon3: Get \"https://packages.vmware.com/photon/photon_oval_definitions/com.vmware.phsa-photon3.xml\": x509: certificate signed by unknown authority\nsuse-updater-opensuse.leap.15.1: Get \"https://support.novell.com/security/oval/opensuse.leap.15.1.xml\": x509: certificate signed by unknown authority\nsuse-updater-suse.linux.enterprise.server.15: Get \"https://support.novell.com/security/oval/suse.linux.enterprise.server.15.xml\": x509: certificate signed by unknown authority\nsuse-updater-suse.linux.enterprise.server.11: Get \"https://support.novell.com/security/oval/suse.linux.enterprise.server.11.xml\": x509: certificate signed by unknown authority\nsuse-updater-suse.linux.enterprise.server.12: Get \"https://support.novell.com/security/oval/suse.linux.enterprise.server.12.xml\": x509: certificate signed by unknown authority\nsuse-updater-opensuse.leap.15.0: Get \"https://support.novell.com/security/oval/opensuse.leap.15.0.xml\": x509: certificate signed by unknown authority\ndebian-buster-updater: failed to retrieve OVAL database: Get \"https://www.debian.org/security/oval/oval-definitions-buster.xml\": x509: certificate signed by unknown authority\n","time":"2021-09-19T14:36:30Z","message":"errors encountered during updater run"}

**Comment by hdonnay** [comment, marked as answer]
It does look like there's something wrong with your setup's PKI. If Clair is unable to fetch any vulnerabilities, it will not have any to match against.
